### PR TITLE
Add a step to ensure velum is ready before bootstrapping

### PR DESCRIPTION
**Pull request by lcavajani**
This is especially required for hyper-v where the Velum page takes minimum 45s to be up.

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

### CODE REVIEW

File: vars/bootstrapEnvironment.groovy [hunk 1]
```diff
@@ -19,6 +19,12 @@ Environment call(Map parameters = [:]) {
 
     // TODO: This and configureEnvironment share 90% of the same code
 
+    timeout(125) {
+        dir('automation/misc-tools') {
+            sh(script: "python ./wait-for-velum https://$(jq '.minions[0].addresses.publicIpv4' -r ${WORKSPACE}/environment.json) --timeout 2")
```
Comment (MalloZup) [inline vars/bootstrapEnvironment.groovy]: is this python3 or 2.7? we might use the right one bin

Comment (lcavajani) [reply vars/bootstrapEnvironment.groovy]: works with both but the CI runs on opensuse 15 ?

**Comment by MalloZup** [comment]
ok feel free to merge once ci is green

**Comment by MalloZup** [comment]
@tdaines42  feel free to review and merge if is ok for you thx

**Comment by tdaines42** [comment]
@lcavajani Just squash the commits and I'll merge it

**Comment by MalloZup** [comment]
:tennis:

**Comment by tdaines42** [comment]
Oh looks like I might be able to do that I'll give it a try

**Pull request merged**